deptrac.yaml

```diff
@@ -85,6 +85,10 @@ deptrac:
               value: ^GuzzleHttp\\*
             - type: className
               value: ^Buzz\\*
+      -   name: RamseyUuid
```

Comment:
indentation looks wonky here... tabs vs spaces?

Reply:
indentation in this file is strange, made this and the previous section match the other `layers` children.